homeassistant/components/shopping_list/__init__.py

```diff
@@ -15,17 +15,21 @@
 from .const import DOMAIN
 
 ATTR_NAME = "name"
+ATTR_COMPLETE = "complete"
 
 _LOGGER = logging.getLogger(__name__)
 CONFIG_SCHEMA = vol.Schema({DOMAIN: {}}, extra=vol.ALLOW_EXTRA)
 EVENT = "shopping_list_updated"
-ITEM_UPDATE_SCHEMA = vol.Schema({"complete": bool, ATTR_NAME: str})
+ITEM_UPDATE_SCHEMA = vol.Schema({ATTR_COMPLETE: bool, ATTR_NAME: str})
 PERSISTENCE = ".shopping_list.json"
 
 SERVICE_ADD_ITEM = "add_item"
 SERVICE_COMPLETE_ITEM = "complete_item"
-
+SERVICE_UNCOMPLETE_ITEM = "uncomplete_item"
```

Comment:
Might be more grammatically sound to rename this to incomplete instead of uncomplete. As I write this, Grammarly is trying to autocorrect my spelling :laughing: